tests/test_dev.py

```diff
@@ -89,39 +88,38 @@ def classmethod_b(cls):
             assert TestClass_deprecationwarning().classmethod_b() == "b"
 
     def test_deprecated_deadline(self):
-        @deprecated(deadline=(2000, 1, 1))
-        def func_old():
-            pass
-
         with warnings.catch_warnings(record=True) as warn_msgs:
-            # Trigger a warning.
-            func_old()
-            # Verify message
-            assert "will be removed on 2000-01-01" in str(warn_msgs[0].message)
+
+            @deprecated(deadline=(2000, 1, 1))
+            def func_old():
+                pass
+
+            assert "This function should have been removed" in str(warn_msgs[0].message)
```

Comment:
I would need to patch this as well. Because currently the code is being pushed from my fork, and as such it is not "in code owner" repo, therefore no warning would be issue. And this is [passing in my fork](https://github.com/DanielYang59/monty/actions/runs/8497007313/job/23275096376), could confirm my assumption

Reply:
that should be easy, no? `monkeypatch.setenv("GITHUB_REPOSITORY", "materialsvirtuallab/monty")`

Reply:
> `monkeypatch.setenv("GITHUB_REPOSITORY", "materialsvirtuallab/monty")`

This is a good fix, but I'm thinking if we could make it better:
- Somehow patch `_is_in_owner_repo` to always return `True` (I'm still experimenting on this) such that we don't need to hard code the repo name (`"materialsvirtuallab/monty"`). Even when we hard code the repo name, unit test would still fail in forks.
- Set the `CI` variable as well, such that unit test wouldn't fail locally.

Reply:
I experimented on patching `_is_in_owner_repo` and did have much luck, then I decide to patch  `subprocess` instead (bcc50019cd101458f0dc211d70a1e1c143b1e700):
```python
with patch("subprocess.run") as mock_run:
                # Mock "GITHUB_REPOSITORY"
                monkeypatch.setenv("GITHUB_REPOSITORY", "TESTOWNER/TESTREPO")
                mock_run.return_value.stdout.decode.return_value = (
                    "git@github.com:TESTOWNER/TESTREPO.git"
                )
```
And everything is working as expected so far.